### ISSUE TITLE
Where are the training and test sets that you used in your article?

### ISSUE THREAD

**Issue by UnixJunkie** [jkwang93/Atom-Path-Descriptor-based-machine-learning#2]
Dear colleagues,
Where are the 3D conformers with assigned partial charges that you used in your paper?
There should be a training set and a test set.

If available, I will cite your paper for sure.

Best regards,
Francois.

**Comment by UnixJunkie** [comment]
Not publishing datasets seriously hinders the reproducibility of science.

**Comment by jkwang93** [comment]
In the "Data Collection" of the paper, we reported the dataset information, please read carefully.
The dataset were colection in the paper "Bleiziffer, P., Schaller, K. and Riniker, S. Machine Learning of Partial Charges Derived From High-Quality Quantum-Mechanical Calculations. Journal of Chemical Information & Modeling 2018;58(3):acs.jcim.7b00663". 
you can get the dataset in this paper.

**Comment by UnixJunkie** [comment]
Ok, nice, thanks!
